Assets/HoloToolkit/Utilities/Editor/BuildCommands.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Implements functionality for building HoloLens applications
+    /// </summary>
+    public static class BuildCommands
+    {
+        public static readonly string BuildLocation = "Builds/HoloLens";
```

Comment:
Consider naming this App since that's what is most common in all the Holographic Academy tutorials.